SourceCode/h5/triangle2.cpp

```diff
@@ -0,0 +1,226 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <cmath>
+
+class Vector {
+    float x, y, z;
```

Comment:
It's good to be able to index into the vertices, so storing them in an array could be better.